libmscore/layout.cpp

```diff
@@ -3505,6 +3512,12 @@ System* Score::collectSystem(LayoutContext& lc)
                   }
             }
       system->layout2();   // compute staff distances
+      // Layout range annotation spanners now, since staff distances have been calculated
+      if (rangeAnnotations.size() > 0) {
+            for (Spanner* s : rangeAnnotations) {
```

Comment:
be sure to review the coding style guidelines.  no braces for blocks of a single statement